pom.xml

```diff
@@ -174,8 +184,8 @@
   </scm>
 
   <build>
-
     <plugins>
+
```

Comment:
Personally, I would remove the whitespace between these definitions I don't think it makes the code any more readable

Reply:
Personally, I prefer this whitespace as it increases readability for me.

Reply:
@julianladisch Ok, thank you for the feedback